sim/firesim-lib/src/main/cc/bridges/simplenic.cc

```diff
@@ -19,7 +19,7 @@
 
 #define SIMLATENCY_BT (this->LINKLATENCY / TOKENS_PER_BIGTOKEN)
 
-#define BUFWIDTH (512 / 8)
+#define BUFWIDTH BridgeConstants::STREAM_WIDTH_BYTES
```

Comment:
Ideally these preprocessor macros would all become constexpr things but I can see how we're going little-by-little.